packages/back-end/src/scripts/generate-openapi.mjs

```diff
@@ -36,14 +36,14 @@ async function run() {
   // Step 3: Add additional named types for easier access
   // Export each schema as a named type
   output += `import { z } from "zod";\n`;
-  output += `import * as openApiValidators from "back-end/src/validators/openapi";\n`;
+  output += `import * as openApiValidators from "../src/validators/openapi";\n`;
```

Comment:
Not sure about this change. This isn't an import within the scripts directory. It's generating a file that will live in types. I checked the other files in types and they use absolute imports.

Reply:
good point